ckanext/data_qld_theme/fanstatic/next-update-due.js

```diff
@@ -0,0 +1,36 @@
+jQuery(document).ready(function () {
```

Comment:
Add some general comments to help explain the business logic and reasons
Remove extra blank lines etc
Run lint and format the code